src/action.py

```diff
@@ -11,6 +11,8 @@ def main():
     parser.add_argument('hashref', help='Hash value of commit to compare against (reference commit)')
     parser.add_argument('hashcomp', help='Hash value of commit to compare')
     parser.add_argument('output_path', help='Path of folder to output the generated page')
+    parser.add_argument('repository_url', help='URL of the tested code repository (for commit hyperlinks)')
```

Comment:
There is no sense in appending a local commit hash to another repo URL

Reply:
I propose this goes in a new issue. I created #35 for it.